docs/develop/localization.md

```diff
@@ -145,6 +145,195 @@ For Outlook add-ins, the [SourceLocation] element also aligns to the form factor
 </PhoneSettings>
 ```
 
+## Localize extended overrides
+
+Some extensibility features of Office Add-ins, such as keyboard shortcuts, are configured with JSON files that are hosted on your server, instead of with the add-in's XML manifest. This section assumes that you are familiar with extended overrides. See [Work with extended overrides of the manifest](extended-overrides.md) and [ExtendedOverrides](../reference/manifest/extendedoverrides.md) element.
+
+There are two ways to localize the strings and URLs in an extended overrides file.
+
+### Method 1: Use resource tokens in the extended overrides file
+
+We recommend this method as the most efficient for most add-ins. You can use the `ResourceUrl` attribute of the [ExtendedOverrides](../reference/manifest/extendedoverrides.md) element to point Office to a file of localized resources. The following is an example.
```

Comment:
```suggestion
We recommend this method as the most efficient for most add-ins. Use the `ResourceUrl` attribute of the [ExtendedOverrides](../reference/manifest/extendedoverrides.md) element to point Office to a file of localized resources. The following is an example.
```

Reply:
Here again, I want to keep the "You can" because it is not mandatory to do this. Later in the article, another way to localize is described.

Reply:
This is not in conformance. Either we recommend or we don't.